code/computational_geometry/src/chans_algorithm/chans_algorithm.cpp

```diff
@@ -0,0 +1,249 @@
+    #include <iostream>
+    #include <stdlib.h>
+    #include <vector>
+    #include <algorithm>	// For qsort() algorithm
+    #include <utility>		// For pair() STL
+    #define RIGHT_TURN -1  // CW
+    #define LEFT_TURN 1  // CCW
+    #define COLLINEAR 0  // Collinear
+    using namespace std;
```

Comment:
What's with the indentation here? Also, avoid `using namespace std` at all costs.